docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -524,7 +524,7 @@ describe('Home screen', () => {
 <Collapsible summary="e2e/utils/openApp.js (new file)">
 
 ```js e2e/utils/openApp.js
-const appConfig = require('../../../app.json');
+const appConfig = require('../../app.json');
```

Comment:
From `e2e/utils/openApp.js` we only need to traverse back 2 levels to get back to the root of the project where app.json lives.